django-ory-cloud/test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This version is more compatible (works on more machines) and generally recommended:

```suggestion
#!/usr/bin/env bash
```

Background: The old version works only if the `bash` executable exists at `/bin/bash`. This is often the case but not always. Some machines might not ship with Bash out of the box and people install it somewhere else. Or machines ship with an outdated version of Bash and the user installs a newer version somewhere else. Our scripts should use the `bash` that the user put in the PATH, no matter in which folder it is.

Another advantage is that you can use any executable here, for example:

```js
#/usr/bin/env node

console.log("This is javascript code running as an executable file!")
```